### PR TITLE
ICDC-730 Implementing Program Cards component

### PR DESCRIPTION
**Pull request by skiran86**

### CODE REVIEW

File: src/main/frontend/src/pages/programs/components/programcard.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
+/* eslint-disable */
```
Comment (AjayDoddapaneni) [inline src/main/frontend/src/pages/programs/components/programcard.js]: Remove lint-disable and update the PR

File: src/main/frontend/src/pages/programs/components/programcard.js [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
 import React from 'react';
 import { withStyles } from '@material-ui/core/';
 import Card from '@material-ui/core/Card';
 import CardMedia from '@material-ui/core/CardMedia';
 import CardContent from '@material-ui/core/CardContent';
 import CardActions from '@material-ui/core/CardActions';
-import cotcImg from '../../../assets/cotc.png';
+import Badge from '@material-ui/core/Badge';
+// eslint-disable-next-line no-unused-vars
```
Comment (AjayDoddapaneni) [inline src/main/frontend/src/pages/programs/components/programcard.js]: These can go now

File: src/main/frontend/src/pages/programs/components/programcard.js [hunk 3]
```diff
@@ -29,28 +36,41 @@ const styles = (theme) => ({
   },
 });
 
+const getFile = (program_acronym) => {
```
Comment (AjayDoddapaneni) [inline src/main/frontend/src/pages/programs/components/programcard.js]: Rename to appropriate func name.

File: src/main/frontend/src/pages/programs/components/programcard.js (hunk 1 above)
Comment (skiran86) [reply src/main/frontend/src/pages/programs/components/programcard.js]: Done

File: src/main/frontend/src/pages/programs/components/programcard.js (hunk 3 above)
Comment (skiran86) [reply src/main/frontend/src/pages/programs/components/programcard.js]: Done

**Pull request merged**